otwcode/otwarchive
# AO3-6392 Comment actions when replying in full page

# Pull Request Checklist


## Issue

https://otwarchive.atlassian.net/browse/AO3-6392

## Purpose

As per the JIRA, fix extra actions showing up and the Cancel button being "stuck" when answering a comment in full page.

## Testing Instructions

Follow the JIRA instructions.

At the end, also try to "Cancel" the reply in the new page. ~~The page should reload instead of the wrong buttons sticking around.~~ The comment box should now collapse properly, the button "Reply" shows up, and hitting it open the box once again.

## Credit

Ceithir (he/him)
